services/Skyra.Database/Models/Entities/Guild.cs

```diff
@@ -68,25 +68,49 @@ public class Guild
 		[StringLength(19)]
 		public string? ChannelsSpam { get; set; }
 
-		[Column("channels.logs.member")]
+		[Column("channels.logs.moderation")]
 		[StringLength(19)]
-		public string? ChannelsLogsMember { get; set; }
+		public string? ChannelsLogsModeration { get; set; }
 
-		[Column("channels.logs.message")]
+		[Column("channels.logs.image")]
 		[StringLength(19)]
-		public string? ChannelsLogsMessage { get; set; }
+		public string? ChannelsLogsImage { get; set; }
 
-		[Column("channels.logs.moderation")]
+		[Column("channels.logs.member-add")]
 		[StringLength(19)]
-		public string? ChannelsLogsModeration { get; set; }
+		public string? ChannelsLogsMemberAdd { get; set; }
 
-		[Column("channels.logs.nsfw-message")]
+		[Column("channels.logs.member-remove")]
 		[StringLength(19)]
-		public string? ChannelsLogsNsfwMessage { get; set; }
+		public string? ChannelsLogsMemberRemove { get; set; }
 
-		[Column("channels.logs.image")]
+		[Column("channels.logs.member-nickname-update")]
 		[StringLength(19)]
-		public string? ChannelsLogsImage { get; set; }
+		public string? ChannelsLogsMemberNickNameUpdate { get; set; }
+
+		[Column("channels.logs.member-username-update")]
+		[StringLength(19)]
+		public string? ChannelsLogsMemberUserNameUpdate { get; set; }
+
+		[Column("channels.logs.member-role-update")]
+		[StringLength(19)]
+		public string? ChannelsLogsMemberRoleUpdate { get; set; }
```

Comment:
```suggestion
		[Column("channels.logs.member-roles-update")]
		[StringLength(19)]
		public string? ChannelsLogsMemberRolesUpdate { get; set; }
```

See other related comment in migration